contrib/release/pull-docker-manifests.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 Authors of Cilium
+
+DIR=$(dirname $(readlink -ne $BASH_SOURCE))
+source $DIR/lib/common.sh
+
+CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}
+IMAGES=(cilium clustermesh-apiserver docker-plugin hubble-relay operator operator-generic operator-aws operator-azure)
+REGISTRIES=(docker.io quay.io)
+
+usage() {
+    logecho "usage: $0 <VERSION>"
+    logecho "VERSION    Target version"
+    logecho
+    logecho "--help     Print this help message"
+}
+
+handle_args() {
+    if ! common::argc_validate 2; then
+        usage 2>&1
+        common::exit 1
+    fi
+
+    if [[ "$1" = "--help" ]] || [[ "$1" = "-h" ]]; then
+        usage
+        common::exit 0
+    fi
+
+    if ! echo "$1" | grep -q "[0-9]\+\.[0-9]\+\.[0-9]\+"; then
+        usage 2>&1
+        common::exit 1 "Invalid VERSION ARG \"$1\"; Expected X.Y.Z"
+    fi
+}
+
+main() {
+    handle_args "$@"
+
+    local ersion="$(echo $1 | sed 's/^v//')"
+    local version="v$ersion"
+
+    >&2 echo n "Fetching docker images for $version"
+    for image in ${IMAGES[@]}; do
+        for registry in ${REGISTRIES[@]}; do
+            >&2 $CONTAINER_ENGINE pull $registry/cilium/$image:$version
```

Comment:
This does basically `docker pull` which can take quite some time.

We might be able to instead ping some API that will tell us the digest? Somewhere under here?

```
https://index.docker.io/v1/repositories/${repo}/tags/${tag}"
```

Reply:
Looks like it, will iterate on this:

```
$ curl --silent -f -lSL "https://hub.docker.com/v2/repositories/cilium/cilium/tags/v1.7.13" | jq -r '.images[] as $i | "docker.io/cilium/cilium:\($i.digest)"'
docker.io/cilium/cilium:sha256:ce1254db59d1113e80ca13748ec54204183e15996efdef0dee915eb69e0b3920
```

Reply:
Eh, I can't seem to figure out a convenient way to pull this from quay.io by tag so I'll just defer for now.